### ISSUE TITLE
State Management Couchbase Components, Deserialize Problem

### ISSUE THREAD

**Issue by yavuzturk96** [dapr/components-contrib#518]
Im using State Management with Couchbase. When i was use dotnet-sdk with couchbase, i cant convert my json state to my model. Redis or other states works great but in couchbase im facing this problem.

#### My Couchbase Yaml
```yaml
apiVersion: dapr.io/v1alpha1
kind: Component
metadata:
   name: statestore
   namespace: default
 spec:
   type: state.couchbase
   metadata:
   - name: couchbaseURL
     value: ****
   - name: username
     value: ****
   - name: password
     value: ****
   - name: bucketName
     value: ****
 ```
#### Dotnet-sdk Version
```
<PackageReference Include="Dapr.AspNetCore" Version="0.11.0-preview01" />
```

#### Sample Http Request:
`http://localhost:<dapr-port>/v1.0/state/statestore/productState`

#### Response:
``` 
\"{\\\"Id\\\":\\\"1\\\",\\\"Name\\\":\\\"T-Short\\\",\\\"Color\\\":\\\"Blue\\\"}\"
```

#### Error
```
await _client.SaveStateAsync<Product>("productState", productState);
var state = await _client.GetStateAsync<Product>("productState");
//Error: System.Text.Json.JsonException: The JSON value could not be converted to Models.ViewModel.Product.
```

**Comment by yaron2** [comment]
We are starting the work to fix this.

**Comment by yaron2** [comment]
@yavuzturk96 this is now fixed and will come out in 1.0-rc2 next week.
